dask-gateway-dev.yaml

```diff
@@ -0,0 +1,57 @@
+# OPTIONAL conda environment file to help get setup with
+# dependencies to build + test dask-gateway locally
+name: dask-gateway-dev
+channels:
+  - conda-forge
+dependencies:
+  # python and pip
+  - python=3.9
+  - pip  
+  
+  # Dependencies to build dask-gateway and dask-gateway-server
+  # see: dask-gateway/setup.py and
+  # dask-gateway-server.setup.py
```

Comment:
Anything duplicated that doesn't clearly describe how to keep things up to date in the two places - gets out of date.

I think for this addition to not be unsustainable, we must have a reference also in those files to link back and say that if what's needed there change, we must change it here as well.